### PR TITLE
translated 'help', 'strings' and 'about' to esperanto

### PR DESCRIPTION
**Pull request by Keksoj**
Created app/src/main/res/values-eo/ directory and all three translated files

### CODE REVIEW

**Review by erciccione — CHANGES_REQUESTED** [review]
Looks like there is at least a duplicated string ("send_address_hint"). You were talking about 10 strings that you added because missing. Please make sure they were actually missing, probably only their value changed.

**Comment by Keksoj** [comment]
The duplicated string mistake and the 10 extra strings come from the https://github.com/m2049r/xmrwallet/commit/39d048fd5e4f177daee298aad16c67187dd6a98b commit, to which I had not updated. 
Fixed it.

File: app/src/main/res/values-eo/about.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <h2>Permesoj de la aplikaĵo</h2>
+        <ul>
+        <li>INTERRETO : Konekti kun la Monero-reto per noda demono</li>
+        <li>LEGI_EKSTERAJN_MEMORILOJN : Legi la monujdosierojn kiuj konserviĝas en la aparato</li>
+        <li>SKRIBI_EKSTERAJN_MEMORILOJN : Skribi la monujdosierojn kiuj konserviĝas en la aparato</li>
+        <li>LASI_VEKA : Lasi la aparaton veka dum la sinkronizado</li>
+        <li>FILMILO : Skani QR-kodojn por ricevi Monerojn</li>
+        </ul>
+        <h2>Ŝanĝoj de la privateca politiko</h2>
+        <p>Ni eble ĝisdatigos tiun privatecan politikon de temp' al tempo. Ni
```
Comment (m2049r) [inline app/src/main/res/values-eo/about.xml]: apostrophes need to be escaped - either as `\'` or as `&apos;`

**Review by m2049r — CHANGES_REQUESTED** [review]
please fix the apostrophe issue

**Comment by Keksoj** [comment]
done

**Review by erciccione — APPROVED** [review]
Looks good to me. Thanks @Keksoj

**Review by m2049r — APPROVED** [review]
Thanks!

**Pull request merged**